ush/parsing_model_configure_FV3.sh

```diff
@@ -38,7 +38,7 @@ write_nsflip:            ${WRITE_NSFLIP:-".false."}
 num_files:               ${NUM_FILES:-2}
 filename_base:           'atm' 'sfc'
 output_grid:             $OUTPUT_GRID
-output_file:             $OUTPUT_FILETYPES
+output_file:             '$OUTPUT_FILETYPE_ATM' '$OUTPUT_FILETYPE_SFC'
```

Comment:
These two settings are resolution dependent, "parallel_netcdf" is not recommended to lower resolution than C384L127 as it takes longer. For C384L127, the "parallel_netcdf" runs faster for atmf files, but not for sfcf file for C768L127, it is recommended to used "parallel_netcdf" for both atmf and sfcf files. If the workflow will also be used by low resolution runs, especially long forecast run, then using "netcdf" will be a better choice.

Reply:
@junwang-noaa 
We are choosing to defining these variables based on the resolution in `config.fv3`. 
There is a `default` also set at the top there.